libs/ledger-live-common/src/families/tron/bridge.integration.test.ts

```diff
@@ -129,12 +129,13 @@ const tron: CurrenciesData<Transaction> = {
             resource: undefined,
             votes: [],
           }),
-          expectedStatus: {
-            amount: new BigNumber("9906000"),
-            errors: {},
-            warnings: {},
-            totalSpent: new BigNumber("10006000"),
-            estimatedFees: new BigNumber("100000"),
+          expectedStatus: (account, transaction, status) => {
+            return {
+              amount: account.spendableBalance.minus(status.estimatedFees),
```

Comment:
Not sure if it's okay to add that much logic in integration tests

Reply:
I'm afraid that fees can move and make our test mark as false negative

Reply:
Ok then :+1: